packages/discord.js/src/errors/ErrorCodes.js

```diff
@@ -12,25 +12,6 @@
  * @property {'TokenMissing'} TokenMissing
  * @property {'ApplicationCommandPermissionsTokenMissing'} ApplicationCommandPermissionsTokenMissing
 
- * @property {'WSCloseRequested'} WSCloseRequested
- * <warn>This property is deprecated.</warn>
- * @property {'WSConnectionExists'} WSConnectionExists
- * <warn>This property is deprecated.</warn>
- * @property {'WSNotOpen'} WSNotOpen
- * <warn>This property is deprecated.</warn>
- * @property {'ManagerDestroyed'} ManagerDestroyed
- * <warn>This property is deprecated.</warn>
-
- * @property {'BitFieldInvalid'} BitFieldInvalid
-
- * @property {'ShardingInvalid'} ShardingInvalid
- * <warn>This property is deprecated.</warn>
- * @property {'ShardingRequired'} ShardingRequired
- * <warn>This property is deprecated.</warn>
- * @property {'InvalidIntents'} InvalidIntents
- * <warn>This property is deprecated.</warn>
- * @property {'DisallowedIntents'} DisallowedIntents
- * <warn>This property is deprecated.</warn>
  * @property {'ShardingNoShards'} ShardingNoShards
```

Comment:
Again, can't comment on the deleted lines but `BitFieldInvalid` wasn't deprecated and is in fact still being used.

Additionally, every other code above this line is still in the javascript array (below), and still have a Message set.